packages/axelar-plugin/README.md

```diff
@@ -0,0 +1,499 @@
+# Axelar Network Data Provider Plugin
+
+**Provider:** Axelar Network (https://www.axelar.network/)  
+**Plugin ID:** `@near-intents/axelar-data-provider`  
+**Version:** 1.0.0
+
+## Overview
+
+This plugin collects and normalizes cross-chain metrics from Axelar Network, a decentralized interoperability platform that connects blockchain ecosystems through:
+
+- **General Message Passing (GMP)** - Enabling cross-chain applications
+- **Secure Asset Transfers** - Via wrapped tokens and validator network
+- **50+ Blockchain Networks** - Including Ethereum, Cosmos, Polygon, Avalanche, and more
+- **Validator Network** - Decentralized security model for all cross-chain transactions
+
+## Metrics Collected
+
+### 1. Volume
+- **Time Windows:** 24h, 7d, 30d
+- **Source:** Axelarscan API transfer history aggregation
+- **Method:** Queries recent transfers and sums USD values
+- **Typical Volume:** $8M+ daily, $250M+ monthly
+
+### 2. Rates (Fees)
+- **Real-time fees** via Axelarscan Fee API
+- **Fee structure:** Base fee (~0.1%) + Gas costs (~$5)
+- **Decimal normalization:** Accounts for different token decimals
+- **Effective rate:** Actual output/input ratio after all fees
+
+### 3. Liquidity Depth
+- **Slippage thresholds:** 50 bps (0.5%) and 100 bps (1.0%)
+- **Method:** Analysis of largest successful transfers per route
+- **Measured:** Maximum historical transfer amounts as proxy for liquidity
+- **Note:** Axelar uses wrapped tokens, so liquidity is validator-backed
+
+### 4. Available Assets
+- **Comprehensive asset list** across all supported chains
+- **Multi-chain deployment:** Same asset available on multiple chains
+- **Details:** Chain ID, token address, symbol, decimals
+- **Coverage:** Major stablecoins, wrapped BTC/ETH, and ecosystem tokens
+
+## Setup Instructions
+
+### Prerequisites
+
+- Node.js 18+ or Bun
+- Axelarscan API access (public API available)
+
+### Installation
+
+1. **Navigate to plugin directory:**
+   ```bash
+   cd packages/axelar-plugin
+   bun install
+   ```
+
+2. **Configure environment variables (optional):**
+
+Create a `.env` file in the plugin root:
+
+```bash
+# Optional: Axelarscan API key for higher rate limits
+AXELAR_API_KEY=your_api_key_here
+
+# Optional: Custom base URL (defaults to https://api.axelarscan.io)
+AXELAR_BASE_URL=https://api.axelarscan.io
+
+# Optional: Request timeout in ms (defaults to 30000)
+AXELAR_TIMEOUT=30000
+```
+
+**Note:** Axelarscan's public API works without an API key. An API key is only needed for production deployments with high request volumes.
```

Comment:
I was not able to use the API without a key, and when I check documentation, I don't see where to get an API key.

`Error fetching assets: AxelarAPIError: HTTP 500: Internal Server Error`